Services/DataBaseService/DbModels/DbUser.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+
+namespace DataBaseService.DbModels
+{
+    public class DbUser
+    {
+        public Guid Id { get; set; }
+        public string Email { get; set; }
+        public string Password { get; set; }
+    }
+
+    public class UserConfiguration : IEntityTypeConfiguration<DbUser>
```

Comment:
Are they at the same file? Why?

Reply:
It`s configuration for user model. Spartak gave advice to put this in same file with model